GetIntoTeachingApiTests/Services/StoreTests.cs

```diff
@@ -500,10 +533,7 @@ private static IEnumerable<TeachingEvent> MockTeachingEvents()
                 TypeId = (int)TeachingEvent.EventType.TrainToTeachEvent,
                 IsOnline = true,
                 StartAt = DateTime.UtcNow.AddDays(5),
-                Building = new TeachingEventBuilding()
-                {
-                    Id = sharedBuildingId,
-                }
+                BuildingId = sharedBuildingId

```

Comment:
Instead of having all the `BuildingId`s as constants can we just have something like:

```
var buildings = MockTeachingEventBuildings();
var sharedBuildingId = buildings[0];

...
BuildingId = buildings[1]
...
BuildingId = buildings[2]
```